lang_php/analyze/ast_php.ml

```diff
@@ -169,6 +169,7 @@ type expr =
    * The string contains the '$'.
   *)
   | Var of var
+  | Metavar of var (* sgrep-ext *)
```

Comment:
You need to add the comment about why we need that in PHP whereas we just hijack Id in the other languages.

Reply:
An alternative is to keep Var, like before, but use at more places the check AST_generic_helpers.is_metavar_name
to distinguish the two, especially if we are in Flag_parsing.sgrep_mode